### ISSUE TITLE
Focus ring is not visbile for the show modal buttons (angular)

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4599]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 2.4.7 Focus Visible

**Test Environment:**
- Test URL: https://clarity.design/documentation/modals
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 19.3.1

**Steps to reproduce:**
1. Open the test URL.
2. Move focus onto to the first Show Modal button present under Angular modals and observe if focus ring around the button is visible and has color contrast ratio of 3:1.

**Expected result:**
Focus ring around the button should be visible and it should have color contrast ratio of 3:1.
Note: Maintaining at least minimum contrast ratio will also cover 2.4.7 Focus visible (Enhanced).

**Actual result:**
Focus ring is not visible. **Important Note:** Similar issue is also observed for other clarity components in general where focus ring is not clearly visible.

**User Impact:**
If focus ring is not clear keyboard user might feel lost on the page and they might have to do a guess work or navigate back and forth and closely check where the actual focus is. This will add extra navigation steps and it will slow down user while performing the intended activity on the page.

**Reporter Email:** gkhadke@vmware.com

**Comment by chlane** [comment]
@gkhadK found that when the user agent's default focus indicator is obscured by the styling of another element, the author needs to supply a custom visual focus indication or remove the styling that is obscuring the visual focus indication. See https://www.w3.org/WAI/WCAG21/Techniques/failures/F78

**Comment by gnomeontherun** [comment]
Hi there, this is an automated message.

In an effort to dedupe and track accessibility reports consistently, this issue is being closed and tracked internally with our Accessibility team. It will still be triaged and addressed as part of ongoing work.
